mypy.ini

```
@@ -8,4 +8,7 @@ disallow_untyped_decorators = True
 ignore_missing_imports = True
 
 [flask.*]
+ignore_missing_imports = True
+
+[pyspark.*]
```

I believe that you need to put a mypy prefix e.a

```suggestion
[mypy-flask.*]
ignore_missing_imports = True

[mypy-pyspark.*]
```